code/headers/ssd1306_oled_buffered.hpp

```diff
@@ -0,0 +1,25 @@
+#include <hwlib.hpp>
+#include <i2c_bus.hpp>
+#include <ssd1306.hpp>
+
+namespace r2d2::display {
+    class ssd1306_oled_buffered : public r2d2::display::ssd1306_i2c_c,
+                                    public hwlib::window {
+    private:
+        static auto constexpr wsize = hwlib::xy(128, 64);
```

Comment:
Why are you keeping an extra variable that is already in hwlib::window?

Reply:
This is because I need the size to form the buffer, and the hwlib::window's size doesn't exist until initialization.